doc/release/1.5.0-notes.rst

```diff
@@ -80,24 +310,813 @@ Backwards incompatible changes
 
 `scipy.linalg` changes
 ----------------------
+The output signatures of ``?syevr``, ``?heevr`` have been changed from 
+``w, v, info`` to ``w, v, m, isuppz, info``
+
+The order of output arguments ``w``, ``v`` of ``<sy/he>{gv, gvd, gvx}`` is 
+swapped.
 
 `scipy.signal` changes
 ----------------------
+The output length of `scipy.signal.upfirdn` has been corrected, resulting 
+outputs may now be shorter for some combinations of up/down ratios and input 
+signal and filter lengths.
+
+`scipy.signal.resample` now supports a ``domain`` keyword argument for
+specification of time or frequency domain input.
 
 `scipy.stats` changes
 ---------------------
 
 
 Other changes
 =============
+Improved support for leveraging 64-bit integer size from linear algebra backends
+in several parts of the SciPy codebase.
+
+Shims designed to ensure the compatibility of SciPy with Python 2.7 have now 
+been removed.
+
+Many warnings due to unused imports and unused assignments have been addressed.
+
+Many usage examples were added to function docstrings, and many input 
+validations and intuitive exception messages have been added throughout the
+codebase.
+
+Early stage adoption of type annotations in a few parts of the codebase
 
 
 Authors
 =======
 
+* @endolith
+* Hameer Abbasi
+* ADmitri +
+* Wesley Alves +
+* Berkay Antmen +
+* Sylwester Arabas +
+* Arne Küderle +
+* Christoph Baumgarten
+* Peter Bell
+* Felix Berkenkamp
+* Jordão Bragantini +
+* Clemens Brunner +
+* Evgeni Burovski
+* Matthias Bussonnier +
+* CJ Carey
+* Derrick Chambers +
+* Leander Claes +
+* Christian Clauss
+* Luigi F. Cruz +
+* dankleeman
+* Andras Deak
+* Milad Sadeghi DM +
+* jeremie du boisberranger +
+* Stefan Endres
+* Malte Esders +
+* Leo Fang +
+* felixhekhorn +
+* Isuru Fernando
+* Andrew Fowlie
+* Lakshay Garg +
+* Gaurav Gijare +
+* GitHub
```

Comment:
user `GitHub` shows up on every release &  I end up deleting manually; should probably figure out if that's leaking from incomplete `.mailmap` or something else